cmake/templates/oneDPLConfig.cmake.in

```diff
@@ -16,7 +16,12 @@
 get_filename_component(_onedpl_root "${CMAKE_CURRENT_LIST_DIR}" REALPATH)
 get_filename_component(_onedpl_root "${_onedpl_root}/../../../" ABSOLUTE)
 
-get_filename_component(_onedpl_headers "${_onedpl_root}/include" ABSOLUTE)
+set(_onedpl_headers_subdir linux)
+if (WIN32)
+    set(_onedpl_headers_subdir windows)
+endif()
```

Comment:
I'm not sure about code style, but there is overwriting of `_onedpl_headers_subdir` on Windows. If it's ok, then everything is alright.

Reply:
Fixed